ckanext/dgu/gemini_postprocess.py

```diff
@@ -151,30 +152,20 @@ def _try_wms_url(url, version='1.3'):
         capabilities_url = wms_capabilities_url(url, version)
         log.debug('WMS check url: %s', capabilities_url)
         try:
-            res = urllib2.urlopen(capabilities_url, None, 10)
-            xml = res.read(MAX_BYTES_READ_DURING_WMS_CHECK + 1)
-        except urllib2.HTTPError, e:
-            # e.g. http://aws2.caris.com/sfs/services/ows/download/feature/UKHO_TS_DS
-            log.info('WMS check for %s failed due to HTTP error status "%s". Response body: %s', capabilities_url, e, e.read())
-            return False
-        except urllib2.URLError, e:
-            log.info('WMS check for %s failed due to HTTP connection error "%s".', capabilities_url, e)
-            return False
+            res = requests.get(capabilities_url)
+            xml = res.text
         except socket.timeout, e:
```

Comment:
I'm not convinced we even need these anymore, but I'm worried about removing them in case it breaks something.